evergreen/manage.py

```diff
@@ -1,9 +1,9 @@
+
```

Comment:
Delete this newline. That actually breaks functionality (the command to tell it to run as python MUST be the first line, otherwise this will get treated as a shell script)

Reply:
done